src/routes/layout.scss

```diff
@@ -0,0 +1,22 @@
+@use "uswds-core" as * with (
+  $theme-show-notifications: false
+);
+
+/* For some reason the border between the header and the rest of the page is 
+    placed on the next sibling of the header, but only if it's a .usa-section
+    or <main>, presuambly since those are full-width elements. Since there's no
+    comments or guidance around this we'll override this rule and apply it to
+    the bottom of the header itself, which seems like a more sane approach.
+   USWDS Source: 
+    https://github.com/uswds/uswds/blob/c397204e35a20cc31e9a2da9567cac8d0214264a/packages/usa-header/src/styles/_usa-header.scss#L58-L64
+*/
+@include at-media("desktop") {
+  .usa-header {
+    border-bottom: 1px solid $base-lighter;
+    // Override with no border on next sibling.
+    & + .usa-section,
+    & + main {
+      border-top: none !important;
```

Comment:
I'd _really_ prefer not to use `!important`, it's a serious maintainability issue. Does placing this CSS at the end of `src/app.scss` override it consistently? It's not perfect since the CSS is a little further away from the code it's affecting but I think it's a lesser evil than `!important`.

Reply:
Good call, I hadn't thought to move it into `src/app.scss`, but this seems to work well without the need for `!important`. Resolved in c31c45b.